src/main.cc

```diff
@@ -24,6 +24,10 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+#ifdef TRITON_ENABLE_PERF_ANALYZER_C_API
```

Comment:
Hah, I hadn't spotted the fact the API is conditional either. This is indeed an added complexity to the shutdown ordering problem.